components/crowdfunding-redesign/queries.ts

```diff
@@ -19,6 +19,92 @@ export const profilePageQuery = gql`
         url
       }
 
+      updates(onlyPublishedUpdates: true) {
+        totalCount
+        nodes {
+          id
+          title
+          html
+          slug
+          summary
+          reactions
+          userReactions
+          userCanSeeUpdate
+          publishedAt
+          fromAccount {
+            id
+            name
+            slug
+            imageUrl
+            type
+          }
+          comments {
+            totalCount
+          }
+        }
+      }
+      expenses(limit: 20, direction: RECEIVED, status: PAID, includeChildrenExpenses: $includeChildren) {
+        totalCount
+        nodes {
+          id
+          description
+          type
+          status
+          createdAt
+          payee {
+            id
+            name
+            type
+            imageUrl
+            slug
+          }
+          account {
+            id
+            name
+            type
+            imageUrl
+            slug
+          }
+          amountV2 {
+            valueInCents
+            currency
+          }
+        }
+      }
+
+      contributionTransactions: transactions(
+        limit: 20
+        kind: [CONTRIBUTION, ADDED_FUNDS]
+        isRefund: false
+        type: CREDIT
+      ) {
+        totalCount
+        nodes {
+          id
+          description
+          createdAt
+          type
+          kind
+          fromAccount {
+            id
+            name
+            type
+            slug
+            imageUrl
+          }
+          toAccount {
+            id
+            name
+            type
+            slug
+            imageUrl
+          }
+          amount {
+            valueInCents
+            currency
+          }
+        }
+      }
```

Comment:
I'd argue that, now that these data are isolated in tabs, we should make the tabs responsible for fetching the data they rely on and keep the main query as light as possible to fasten the collective page load.

Reply:
Definitely, the data loading overall needs more consideration in this prototype.